src/components/MultiLineText.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export const MultiLineText = ({ value }: { value: string }) => {
```

Comment:
There were line breaks (`\n`) in the existing texts which didn't have any effect. Added this simple component just to split texts, without using react i18n `Trans` component everywhere.

Reply:
this shouldn't be done using `<br>` in most cases `<p>` should be used as it is semantically correct.

Reply:
you can also respect `\n` by setting css properties: https://stackoverflow.com/questions/4560038/is-it-possible-to-make-any-css-element-behave-like-pre

Reply:
`<p>`  wasn't working as it was, because it produced nested `<p>` tags.

The css solution wasn't working as described at stackoverflow. But I figured out another plain css solution now. Hope, it's fine now. 🙂